react/components/SearchResultContainer.js

```diff
@@ -1,18 +1,17 @@
 import React, { useState, useRef, useCallback } from 'react'
-import { min } from 'ramda'
+import { min, max } from 'ramda'
 
 import { Container } from 'vtex.store-components'
 
 import { PopupProvider } from './Popup'
-import InfiniteScrollLoaderResult from './loaders/InfiniteScrollLoaderResult'
-import ShowMoreLoaderResult from './loaders/ShowMoreLoaderResult'
 import { searchResultContainerPropTypes } from '../constants/propTypes'
+import { useRuntime } from 'vtex.render-runtime'
 import {
   useSearchPageStateDispatch,
   useSearchPageState,
 } from 'vtex.search-page-context/SearchPageContext'
-
-const PAGINATION_TYPES = ['show-more', 'infinite-scroll']
+import SearchResult from './SearchResult'
+import InfiniteScroll from 'react-infinite-scroll-component'
```

Comment:
⛏ (Nitpick) This import could be moved up to line 3, just to group together library imports.
I'd also group together line 8, 9-12 and 4 since these are imports for VTEX packages.